app/services/host_aspects.rb

```diff
@@ -0,0 +1,11 @@
+module HostAspects
+  class << self
+    def configuration
+      @configuration ||= HostAspects::Configuration.new
+    end
+
+    def configure
```

Comment:
When you want to configure your aspect, it will be in the initializer code:

``` Ruby
HostAspects.configure do |config| 
  config.register 'MyAspect'
```